src/dsp_tools/commands/excel2json/models/input_error_lists.py

```diff
@@ -32,7 +33,27 @@ def execute_error_protocol(self) -> str:
 
 
 @dataclass
-class ListSheetProblem:
+class CollectedSheetProblems:
```

Comment:
- This class is unused in the code.
- If you have unimplemented methods, shouldn't it then be a Protocol? Otherwise it doesn't make sense to even define these methods.
- Not sure if it is usual to `raise NotImplementedError` in an unimplemented Protocol method. I thought it would rather be done with `...`:

```
def execute_error_protocol(self) -> str: ...
```

Reply:
I can delete it, but in the next PR I will add it again...
I prefer to do the not implemented, so that you for sure do not use it directly.

Reply:
But then it should still be a Protocol, shouldn't it?

Reply:
No it will be used directly. But I just did not write the functions yet. I'll delete it.